config/routes.rb

```diff
@@ -149,6 +149,12 @@
     post 'users/:id/close_account' => 'users#close_account', :as => 'close_account'
     post 'users/:id/lock_account' => 'users#lock_account', :as => 'lock_account'
     post 'users/:id/unlock_account' => 'users#unlock_account', :as => 'unlock_account'
+    post 'users/:id/make_admin' => 'users#make_admin', :as => 'make_admin'
+    post 'users/:id/remove_admin' => 'users#remove_admin', :as => 'remove_admin'
+    post 'users/:id/make_moderator' => 'users#make_moderator', :as => 'make_moderator'
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.